package.json

```diff
@@ -55,6 +55,7 @@
     "handlebars": "^4.7.7",
     "honeyswap-sdk": "^1.0.0",
     "html-pdf-node": "^1.0.7",
+    "husky": "^4.3.8",
```

Comment:
How about setting exact version for these dependencies?
(for instance `4.3.8` instead of `^4.3.8` to prevent unwanted upgrades

Reply:
You are right. I think there are no more updates actually. 4.3.8 is the last one. 
Thanks

Reply:
Thanks @CarlosQ96